examples/todo/src/openapi-spec.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright IBM Corp. 2018,2020. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {ApplicationConfig} from '@loopback/core';
+import fs from 'fs';
+import {dump} from 'js-yaml';
+import {TodoListApplication} from './application';
+
+/**
+ * Save the OpenAPI spec to the given file
+ * @param outFile - File name for the spec
+ * @param options - Application config
+ */
+export async function dumpOpenApiSpec(
+  outFile: string,
+  options?: ApplicationConfig,
+) {
+  options = options ?? {};
+  options.rest = {...options.rest, listenOnStart: false};
```

Comment:
Let's not modify the `options` object provided by the caller.

```suggestion
  options = {...options};
  options.rest = {...options.rest, listenOnStart: false};
```